tests/events/src/lib.rs

```diff
@@ -100,25 +87,14 @@ mod test {
         client.transfer(&from, &to, &amount);
 
         assert_eq!(
-            env.events().all(),
```

Comment:
Can we preserve the use of `all` in this test while both fns exist in the sdk? So keep the existing assertion, and then add the new assertion behind it.